duckduckgo/macos-browser
# Require authentication for login export

Task/Issue URL: https://app.asana.com/0/1177771139624306/1202053082503796/f
Tech Design URL:
CC:

**Description**:

This PR adds an authentication step when exporting logins.

**Steps to test this PR**:
1. Enable Autofill authentication in preferences, and then try to export logins - verify that you get an authentication prompt. 
1a. Verify that canceling the authentication prompt exits the import flow, you should get no further windows appearing
1b. Verify that authenticating shows the export window
2. Check that the export window contains a red disclaimer label, and that it is laid out correctly (that is, not truncated in any way)

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
